build/yamls/antrea-octant.yml

```diff
@@ -69,3 +69,5 @@ spec:
         secret:
           defaultMode: 256
           secretName: octant-kubeconfig
+      nodeSelector:
```

Comment:
Remove the nodeSelector here or at least add "#".  It is for testing and antrea-octant pod can run on any node.

Reply:
@mengdie-song Is this file generated?

Reply:
> @mengdie-song Is this file generated?

This file can be updated via "make manifest".